recipe/preload_config/linux64_cuda11.2_wheel.json

```diff
@@ -0,0 +1,7 @@
+{
+ "packaging": "conda",
+ "cuda": "11.2",
+ "cudnn": {"version": "", "filename": ""},
+ "nccl": {"version": "", "filename": ""},
+ "cutensor": {"version": "", "filename": ""}
```

Comment:
What does CuPy do when the version/filename are unspecified?

Reply:
CuPy will not use the contents of the dict if `packaging == 'conda'`.
https://github.com/cupy/cupy/blob/v9.0.0rc1/cupy/_environment.py#L281

But `cudnn`, `nccl`, `cutensor` keys must exist in the json to show an appropriate message.
https://github.com/cupy/cupy/blob/v9.0.0rc1/cupy/_environment.py#L349-L365

Reply:
Thanks Kenichi! 😄

Since they are not used, could these be just `{}`? Or am I missing something?

Reply:
That makes sense, leaving it `{}` sounds better.

Reply:
I will do that here and in the other PR (once the Windows local test is done). I will also remove nccl and cutensor from jsons in this PR, as they are not used in v8 as Kenichi pointed out.

Reply:
In v8 jsons cannot be included for conda-forge packages because v8 preloading does not take care of conda.
https://github.com/cupy/cupy/blob/v8.6.0/cupy/_environment.py#L330

Reply:
shoot I forgot again...well looks like we will stick to the old way for v8.

Reply:
On the bright side, they are used in the rc branch (and later the master branch once v9 is out), so I did't waste time...

Reply:
Json files are removed in 2d1a289.